media/Circletracer/loadfile.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let jsonLoadCheck = 0;
+function openFileSelector(flag) {
+  const input = document.createElement('input');
+  input.type = 'file';
+  input.accept = 'application/json';
+  input.onchange = (event) => {
+    setFileName(event.target.files[0].name, flag);
+    processFile(event.target.files[0], flag);
+  };
+  input.click();
+}
+
+function setFileName(name, flag) {
+  if (flag === '1') {
+    document.querySelector('#first-json-btn').innerHTML = name;
```

Comment:
Could we change `flag` to use `#first-json-btn`, not integer?
Current interger `flag` is too ambiguous and implicit

Reply:
In order to change this, I need to change `index.html`.
Will it be okay to fix it after it's merged?

Reply:
Hmm... if so, could you fix it first while we apply feedbacks to this PR? :)

Reply:
Please fix already merged codes first! 
That would be better for us :)

Reply:
We fixed `index.html` :)

Reply:
Hmm,,, I just checked it out, and I already reflected what you requested. #237

Or is there anything I missed?

Reply:
I finally fixed it properly...sorry T.T